services/workflows-service/src/providers/translation/locales/en/translation.json

```diff
@@ -637,7 +637,7 @@
       "holderName": "Holder name should be at least 5 characters long.",
       "companyAccountHolderName": "Company account holder name should be at least 5 characters long.",
       "holderFullAddress": "Holder full address should be at least 10 characters long.",
-      "accountNumber": "Account number should be at least 6 characters long.",
```

Comment:
Are you sure its the right file? Seems wrong to update characters length for every single customer.